pypesto/optimize/optimize.py

```diff
@@ -15,16 +15,16 @@
 
 
 def minimize(
-        problem: Problem,
-        optimizer: Optimizer = None,
-        n_starts: int = 100,
-        ids: Iterable[str] = None,
-        startpoint_method: Union[Callable, bool] = None,
-        result: Result = None,
-        engine: Engine = None,
-        progress_bar: bool = True,
-        options: OptimizeOptions = None,
-        history_options: HistoryOptions = None,
+    problem: Problem,
+    optimizer: Optimizer = None,
+    n_starts: int = 100,
+    ids: Iterable[str] = None,
+    startpoint_method: Union[StartpointMethod, bool] = None,
```

Comment:
I have to say that I find it a bit counterintuitive to pass a class object to an argument that is called "..._method"